content/en/docs/tasks/administer-cluster/topology-manager.md

```diff
@@ -221,18 +221,29 @@ You can toggle groups of options on and off based upon their maturity level usin
 You will still have to enable each option using the `TopologyManagerPolicyOptions` kubelet option.
 
 The following policy options exists:
-* `prefer-closest-numa-nodes` (beta, visible by default; `TopologyManagerPolicyOptions` and `TopologyManagerPolicyBetaOptions` feature gates have to be enabled).
-The `prefer-closest-numa-nodes` policy option is beta in Kubernetes {{< skew currentVersion >}}.
-
-If the `prefer-closest-numa-nodes` policy option is specified, the `best-effort` and `restricted`
-policies will favor sets of NUMA nodes with shorter distance between them when making admission decisions.
-You can enable this option by adding `prefer-closest-numa-nodes=true` to the Topology Manager policy options.
-By default, without this option, Topology Manager aligns resources on either a single NUMA node or
-the minimum number of NUMA nodes (in cases where more than one NUMA node is required). However,
-the `TopologyManager` is not aware of NUMA distances and does not take them into account when making admission decisions.
-This limitation surfaces in multi-socket, as well as single-socket multi NUMA systems,
-and can cause significant performance degradation in latency-critical execution and high-throughput applications if the
-Topology Manager decides to align resources on non-adjacent NUMA nodes.
+* `prefer-closest-numa-nodes` (beta, and enabled by default).
```

Comment:
I'd change to "beta, and visible by default".

The current unfortunately still not correct. In the context of the options, this means the feature is `visible` by default. Otherwise, you need to enable the feature gate to be able to set the option, and in turn the option default is the backward compatible one.

So it's a 2-level protection:
1. enable the option visibility (default: visible for beta, *not* visible for alpha)
2. enable the option proper

to summarize, the beta options are visible (not enabled) by default.

Sorry for any confusion caused.

Reply:
@ffromani this one I changed by accident and is part of another KEP.